editor.planx.uk/src/@planx/components/Send/Editor.tsx

```diff
@@ -57,6 +57,14 @@ const SendComponent: React.FC<Props> = (props) => {
     },
   ];
 
+  // Show S3 option on staging only
+  if (process.env.REACT_APP_ENV !== "production") {
+    options.push({
+      value: Destination.S3,
+      label: "Upload to AWS S3 bucket",
+    });
+  }
```

Comment:
Rather than introduce a feature-flag here which we'll then have to communicate to Editors how to use - I figure we can just _only_ display this option on non-prod environments at first.

Reply:
Yeah this works well I think 👍